API/Controllers/UserController.cs

```diff
@@ -77,7 +81,7 @@ public async Task<IActionResult> GetCurrentUser()
         /// <summary>
         ///     Get a user account.
         /// </summary>
-        /// <param name="userId"></param>
+        /// <param name="userId">the useridentifier.</param>
         /// <returns></returns>
```

Comment:
Empty returns tag (while this method does return something).
Please, also fix the other empty returns tags in this file :)